src/components/Errors/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Errors';
```

Comment:
Avoid exporting directly from an import. Import the module and then export it separately 📦